pypeit/wavecalib.py

```diff
@@ -628,7 +633,8 @@ def build_wv_calib(self, arccen, method, skip_QA=False,
             # Now preferred
             # Slit positions
             arcfitter = autoid.ArchiveReid(arccen, self.lamps, self.par,
-                                           ech_fixed_format=self.spectrograph.ech_fixed_format, ok_mask=ok_mask_idx,
+                                           ech_fixed_format=self.spectrograph.ech_fixed_format, 
+                                           ok_mask=ok_mask_idx,
                                            measured_fwhms=measured_fwhms,
```

Comment:
could you change here `measured_fwhms` with `self.measured_fwhms`?

Reply:
same below in line 647

Reply:
done